schemas/contributorRole.json

```diff
@@ -0,0 +1,38 @@
+{
+	"title": "ContributorRole",
+	"description": "Contributor role for contributors",
+	"required": [
+		"contextId",
+		"identifier",
+		"name"
+	],
+	"properties": {
```

Comment:
We can also add _href

"_href": {
			"type": "string",
			"origin": "composed",
			"description": "The URL to this contributor role in the REST API.",
			"format": "uri",
			"readOnly": true,
			"apiSummary": true
		},